addons/point_of_sale/static/src/js/models.js

```diff
@@ -1579,8 +1579,8 @@ exports.Product = Backbone.Model.extend({
             return (! item.product_tmpl_id || item.product_tmpl_id[0] === self.product_tmpl_id) &&
                    (! item.product_id || item.product_id[0] === self.id) &&
                    (! item.categ_id || _.contains(category_ids, item.categ_id[0])) &&
-                   (! item.date_start || moment(item.date_start).isSameOrBefore(date)) &&
-                   (! item.date_end || moment(item.date_end).isSameOrAfter(date));
+                   (! item.date_start || moment.utc(item.date_start).isSameOrBefore(date)) &&
+                   (! item.date_end || moment.utc(item.date_end).isSameOrAfter(date));
```

Comment:
I've just tested it and when I do "moment.utc(item.date_end)" it does not seems to do anything. Shouldn't we instead modify "var date = moment()" by "var date = moment().utc() ? Like that we'll be able to compare date which will be in utc with date_end or date_start that are already in UTC.

Reply:
Sorry but when I put a debugger, I get that
![image](https://user-images.githubusercontent.com/20767708/161753933-6d18766e-202f-4971-8c28-4cf0c772ecb5.png)

And this is my rule:
![image](https://user-images.githubusercontent.com/20767708/161754140-2162976f-388d-4631-bae3-4ce00826a440.png)